internal/codegen/tpl_file_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+// Description: This file contains the public API for templates
+// for stencil
+
+package codegen
+
+import (
+	"testing"
+
+	"go.rgst.io/stencil/pkg/slogext"
+	"go.rgst.io/stencil/pkg/stencil"
+)
+
+// TestTplFile_OnceNoLockfile tests the file.Once command when there's no lockfile history at all
+func TestTplFile_OnceNoLockfile(t *testing.T) {
+	tplf := TplFile{
+		f: &File{path: "test.go"},
+	}
+
+	fo, err := tplf.Once()
+	if err != nil {
```

Comment:
**question(non-blocking):** Any reason you're _not_ using an assertion library here, but are in other places? Honestly just curious

Reply:
oh, that was a few days ago before i saw that tehre was an assert library in place in some tests